sim/warlock/talents.go

```diff
@@ -10,29 +10,28 @@ import (
 
 func (warlock *Warlock) ApplyTalents() {
 	// Demonic Embrace
-	if warlock.Talents.DemonicEmbrace > 0 {
-		warlock.AddStatDependency(stats.Stamina, stats.Stamina, 1.01+(float64(warlock.Talents.DemonicEmbrace)*0.03))
-	}
+	warlock.AddStatDependency(stats.Stamina, stats.Stamina, 1.01+(float64(warlock.Talents.DemonicEmbrace)*0.03))
 
-	// Suppression
-	warlock.AddStat(stats.SpellHit, float64(warlock.Talents.Suppression)*core.SpellHitRatingPerHitChance)
+	// Molten Skin
+	warlock.PseudoStats.DamageTakenMultiplier /= 1 + 0.02*float64(warlock.Talents.MoltenSkin)
```

Comment:
Most other talents like this are implemented as `*= 1 - 0.02*float64(warlock.Talents.MoltenSkin)`. Are you sure this is different?